src/shims/unix/sync.rs

```diff
@@ -90,79 +90,120 @@ fn mutex_id_offset<'tcx>(ecx: &MiriInterpCx<'tcx>) -> InterpResult<'tcx, u64> {
     Ok(offset)
```

Comment:
Can you add a FIXME to the sanity check logic above that we should also check the other static initializers for platforms where they exist?

Reply:
Is this still needed after you checked that there are no other other initializers?

Reply:
Yeah it needs to be checked for Linux.